File: examples/comparison/loop/sum_loop.go
package main

import "fmt"

func main() {
	var sum int64 = 0
	const n int64 = 375_000_000
	const mod int64 = 1_000_000_007
	for i := int64(0); i < n; i++ {
		sum = (sum + i) % mod
	}
	fmt.Println(sum)
}

File: examples/comparison/loop/SumLoop.java
public class SumLoop {
    public static void main(String[] args) {
        long sum = 0;
        final long n = 375_000_000L;
        final long mod = 1_000_000_007L;
        for (long i = 0; i < n; i++) {
            sum = (sum + i) % mod;
        }
        System.out.println(sum);
    }
}
